src/EncompassRest/Contacts/BorrowerContacts.cs

```diff
@@ -0,0 +1,171 @@
+                    await response.Content.PopulateAsync(contact).ConfigureAwait(false);
+                }
+                contact.Dirty = false;
+                return Path.GetFileName(response.Headers.Location.OriginalString);
+            });
+        }
+
+        public Task<string> CreateContactRawAsync(string contact) => CreateContactRawAsync(contact, null, CancellationToken.None);
+
+        public Task<string> CreateContactRawAsync(string contact, CancellationToken cancellationToken) => CreateContactRawAsync(contact, null, cancellationToken);
+
+        private Task<string> CreateContactRawAsync(string contact, string queryString) => CreateContactRawAsync(contact, queryString, CancellationToken.None);
+
+        private Task<string> CreateContactRawAsync(string contact, string queryString, CancellationToken cancellationToken)
+        {
+            Preconditions.NotNullOrEmpty(contact, nameof(contact));
+
+            return CreateContactInternalAsync(new JsonStringContent(contact), queryString, cancellationToken, async response =>
+            {
+                var json = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+                return string.IsNullOrEmpty(json) ? Path.GetFileName(response.Headers.Location.OriginalString) : json;
+            });   
+        }
+
+        private async Task<string> CreateContactInternalAsync(HttpContent content, string queryString, CancellationToken cancellationToken, Func<HttpResponseMessage, Task<string>> func)
+        {
+            using (var response = await Client.HttpClient.PostAsync($"{s_apiPath}/{(!string.IsNullOrEmpty(queryString) && queryString[0] != '?' ? "?" : string.Empty)}{queryString}", content, cancellationToken).ConfigureAwait(false))
+            {
+                if (!response.IsSuccessStatusCode)
+                {
+                    throw await RestException.CreateAsync(nameof(CreateContactAsync), response).ConfigureAwait(false);
+                }
+
+                return await func(response).ConfigureAwait(false);
+            }
+        }
+
+        public Task UpdateContactAsync(BorrowerContact contact) => UpdateContactAsync(contact, false, CancellationToken.None);
+
+        public Task UpdateContactAsync(BorrowerContact contact, CancellationToken cancellationToken) => UpdateContactAsync(contact, false, cancellationToken);
+
+        private Task UpdateContactAsync(BorrowerContact contact, bool populate) => UpdateContactAsync(contact, populate, CancellationToken.None);
+
+        private Task UpdateContactAsync(BorrowerContact contact, bool populate, CancellationToken cancellationToken)
+        {
+            Preconditions.NotNull(contact, nameof(contact));
+            Preconditions.NotNull(contact.Id, nameof(contact.Id));
+
+            return UpdateContactInternalAsync(contact.Id, JsonStreamContent.Create(contact), populate ? new QueryParameters(new QueryParameter("view", "entity")).ToString() : null, cancellationToken, async response =>
```

Comment:
Need to initialize contact like at https://github.com/EncompassRest/EncompassRest/blob/master/src/EncompassRest/Loans/Loans.cs#L208